control/control_performance_analysis/launch/controller_performance_analysis.launch.xml

```diff
@@ -1,23 +1,22 @@
 <launch>
 
-  <arg name="control_performance_analysis_param_path"
-       default="$(find-pkg-share control_performance_analysis)/config/control_performance_analysis.param.yaml" />
+  <arg name="control_performance_analysis_param_path" default="$(find-pkg-share control_performance_analysis)/config/control_performance_analysis.param.yaml" />
   <arg name="input/reference_trajectory" default="/planning/scenario_planning/trajectory" />
   <arg name="input/control_raw" default="/control/trajectory_follower/lateral/control_cmd" />
-  <arg name="input/measured_steering" default="/vehicle/status/steering" />
-  <arg name="input/current_velocity" default="/localization/twist" />
+  <arg name="input/measured_steering" default="/vehicle/status/steering_status" />
+  <arg name="input/current_odometry" default="/localization/odometry" />
```

Comment:
```suggestion
  <arg name="input/current_odometry" default="/localization/kinematic_state" />
```

Reply:
Thanks, I changed the topic name. https://github.com/tier4/AutowareArchitectureProposal.iv/pull/698/commits/67e6c9107939d403aa16f9655a0a0a3feaecafa9